src/editor.rs

```diff
@@ -267,32 +267,21 @@ fn parse_command(keymap: &KeyMap, command: &str) -> Option<Action> {
     let mut command_char_iter = command.chars();
 
     // Consume the first char of the command
-    let c = command_char_iter.next()?;
-
-    match keymap.get(&c) {
-        // "q" quits Sapling
-        Some(Command::Quit) => Some(Action::Quit),
-        Some(Command::InsertChild) => {
-            // Consume the second char of the iterator
-            command_char_iter.next().map(Action::InsertChild)
+    return if let Some(command) = command_char_iter.next().and_then(|c| keymap.get(&c)) {
+        match command {
+            // "q" quits Sapling
+            Command::Quit => Some(Action::Quit),
+            Command::InsertChild => command_char_iter.next().map(Action::InsertChild),
+            Command::InsertBefore => command_char_iter.next().map(Action::InsertBefore),
+            Command::InsertAfter => command_char_iter.next().map(Action::InsertAfter),
+            Command::Replace => command_char_iter.next().map(Action::Replace),
```

Comment:
Since we're making the code look pretty, I think it would be more readable if we align all the `=>`s like:
```rust
Command::InsertChild  => command_char_iter.next().map(Action::InsertChild),
Command::InsertBefore => command_char_iter.next().map(Action::InsertBefore),
```

You'll probably have to put `#[rustfmt::skip]` above the `match command {` line to tell rustfmt that we want this the way it is.